lstchain/image/muon/muon_analysis.py

```diff
@@ -177,49 +185,55 @@ def analyze_muon_event(event_id, image, geom, equivalent_focal_length,
          (pix_ring > tailcuts[0]).sum() >
          min_pix_fraction_after_cleaning * min_pix,
          np.count_nonzero(pix_ring) > min_pix,
-         muonringparam.ring_radius < max_ring_radius,
-         muonringparam.ring_radius > min_ring_radius
+         muonringparam.radius < max_ring_radius,
+         muonringparam.radius > min_ring_radius
         ])
 
     if  candidate_clean_ring:
-        intensity_fitter = MuonIntensityFitter()
+        intensity_fitter = MuonIntensityFitter(subarray)
 
         # Use same hard-coded value for pedestal fluctuations as the previous
         # version of ctapipe:
         pedestal_stddev = 1.1*np.ones(len(image))
 
         muonintensityoutput = \
             intensity_fitter(1,
-                             muonringparam.ring_center_x,
-                             muonringparam.ring_center_y,
-                             muonringparam.ring_radius,
+                             muonringparam.center_x,
+                             muonringparam.center_y,
+                             muonringparam.radius,
                              image*dist_mask,
                              pedestal_stddev)
 
-        dist_ringwidth_mask = np.abs(dist - muonringparam.ring_radius) < \
+        dist_ringwidth_mask = np.abs(dist - muonringparam.radius) < \
                               muonintensityoutput.width
+
         # We do the calculation of the ring completeness (i.e. fraction of whole circle) using the pixels
         # within the "width" fitted using MuonIntensityFitter
-        muonintensityoutput.ring_completeness = ring_completeness(
+        muonparameters.completeness = ring_completeness(
             x[dist_ringwidth_mask], y[dist_ringwidth_mask],
                 image[dist_ringwidth_mask],
-            muonringparam.ring_radius,
-            muonringparam.ring_center_x,
-            muonringparam.ring_center_y,
+            muonringparam.radius,
+            muonringparam.center_x,
+            muonringparam.center_y,
             threshold=30,
             bins=30)
 
         pix_ringwidth_im = image[dist_ringwidth_mask]
-        muonintensityoutput.ring_pix_completeness =  \
-            (pix_ringwidth_im > tailcuts[0]).sum() / len(pix_ringwidth_im)
+
+        # No longer existing in ctapipe 0.8:
+        # muonintensityoutput.ring_pix_completeness =  \
+        #     (pix_ringwidth_im > tailcuts[0]).sum() / len(pix_ringwidth_im)
 
     else:
-        # just to have the default values:
+        # just to have the default values with units:
         muonintensityoutput = MuonEfficiencyContainer()
+        muonintensityoutput.width = u.Quantity(np.nan, u.deg)
```

Comment:
Why didn't you open an issue at ctapipe? The defaults should have the correct units

Reply:
already fixed in the next ctapipe version?
https://github.com/cta-observatory/ctapipe/pull/1380